packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -534,11 +540,12 @@ function errorCallback(container: ContainerInfoUI, errorMessage: string): void {
                       name: containerGroup.name,
                       engineId: containerGroup.engineId,
                       engineType: containerGroup.engineType,
-                      containers: [],
+                      containers: containerGroup.containers,
```

Comment:
It makes no sense to send `[]`. With the menu feature, it is very useful to have the list of containers. Therefore adding it them

Reply:
maybe move that to a separate PR as it's not related to this PR